oci/private/tarball.sh.tpl

```diff
@@ -34,5 +35,8 @@ layers="${LAYERS}" \
         --null-input '.[0] = {"Config": env(config), "RepoTags": "${repo_tags}" | envsubst | split("%") | map(select(. != "")) , "Layers": env(layers) | map( "blobs/" + . + ".tar.gz") }' \
         --output-format json > "${STAGING_DIR}/manifest.json"
 
-# TODO: https://github.com/bazel-contrib/rules_oci/issues/217
-tar -C "${STAGING_DIR}" -cf "${TARBALL_PATH}" manifest.json blobs
+# Write output in mtree format
+# https://man.freebsd.org/cgi/man.cgi?mtree(8)
+# so that tar produces a deterministic output.
+echo "manifest.json uid=0 gid=0 mode=0755 type=file content=${STAGING_DIR}/manifest.json">>${OUT}
```

Comment:
Do we want to set also the time for manifest.json and blobs?

Reply:
It would be nice to set the time to something like 2000-01-01 (a second equivalent of that), to ensure that when unpacking the tar we won't get some warnings (for the too old time value).

Reply:
Good idea, output is now

```
$ bazel build examples/repo_tags:tarball
$ tar -tvf bazel-bin/examples/repo_tags/tarball/tarball.tar
-rwxr-xr-x 0/0             123 2023-01-01 00:00 blobs/sha256/dbe00f7d7b6cdecf0837ab201ce929b9cb08fd6960e1fd4e27b076ecbe3e6496
-rw-r--r-- 0/0             234 2023-01-01 00:00 manifest.json
```